src/Tizen.NUI.Components/Style/LoadingStyle.cs

```diff
@@ -58,7 +58,21 @@ public LoadingStyle(LoadingStyle style) : base(style)
         /// Gets or sets loading image size.
         /// </summary>
         /// <since_tizen> 8 </since_tizen>
-        public Size LoadingSize { get; set; }
+        public Size LoadingSize
+        {
+            get
+            {
+                return this.Size;
+            }
+            set
+            {
+                if (value == null)
```

Comment:
I think it is better to allow null value for the style property.

Reply:
Oh I see. Thanks for your review. It's because it's a style property.